include/nccl_ofi_rdma.h

```diff
@@ -51,6 +51,9 @@ extern "C" {
  */
 #define NCCL_OFI_RDMA_SEQ_BITS     (10)
 
+/* Maximum size of eager message */
+extern size_t eager_max_size;
```

Comment:
If we're going to take this approach (and I think https://github.com/aws/aws-ofi-nccl/pull/482 is a simpler solution until we can properly fix this template of global variables everywhere), we need these properly prefixed when naming, so it's obvious that there are side-effecty usages of the variable.

Reply:
Sounds good to merge your patch